sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/BlobDeleteType.java

```diff
@@ -4,47 +4,28 @@
 
 package com.azure.storage.blob.implementation.models;
 
+import com.azure.core.util.ExpandableStringEnum;
 import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonValue;
+import java.util.Collection;
 
-/**
- * Defines values for BlobDeleteType.
- */
-public enum BlobDeleteType {
-    /**
-     * Enum value Permanent.
-     */
-    PERMANENT("Permanent");
-
-    /**
-     * The actual serialized value for a BlobDeleteType instance.
-     */
-    private final String value;
-
-    BlobDeleteType(String value) {
-        this.value = value;
-    }
+/** Defines values for BlobDeleteType. */
+public final class BlobDeleteType extends ExpandableStringEnum<BlobDeleteType> {
```

Comment:
Is it intentional that these are now ExpandableStringEnum?

Reply:
This is related to that other autorest issue I linked to you https://github.com/Azure/autorest.java/issues/918